index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First project in Microverse</title>
+
+    <!-- custom css file link -->
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ]  Kyndly add a class to your h1 tag, its not a good practice to add styles directly to tags.
- example `<h1 class"hello">Hello Microverse!</h1>`

Reply:
I will add it right away thank you so much.